src/main/java/com/checkmarx/flow/controller/GitLabController.java

```diff
@@ -40,6 +40,8 @@
     private static final String EVENT = "X-Gitlab-Event";
     private static final String PUSH = EVENT + "=Push Hook";
     private static final String MERGE = EVENT + "=Merge Request Hook";
+    private static final String MERGE_ID = "merge_id";
```

Comment:
This constant is defined 3 times in the project. It should be defined at a single place.

Reply:
Ok, Using FlowConstants as a single point definition